orderapi/src/test/java/fi/hel/verkkokauppa/order/service/order/OrderServiceTest.java

```diff
@@ -642,6 +649,7 @@ void createOrderWithMerchantId() throws JsonProcessingException {
         order.setPriceTotal(String.valueOf(new BigDecimal(orderItem.getRowPriceTotal())));
         Assertions.assertEquals(firstMerchantIdFromNamespace,orderItem.getMerchantId());
         orderRepository.save(order);
+        ordersToBeDeleted.add(order);
```

Comment:
Do not add this, because I use this for development only. :D

Reply:
add some kind of boolean to class to change via environment variable, I use these alot.

Reply:
Ok, then I will just remove it